action_viafence/viafence_action.py

```diff
@@ -89,7 +97,9 @@ def createVias(self, viaPoints, viaDrill, viaSize, netCode):
         return newVias
 
     def selfToMainDialog(self):
-        self.mainDlg.lstLayer.SetItems(self.layerMap.values())
+        #wx.LogMessage(str(self.layerMap.values()))
```

Comment:
There seem to be some remaining comments for development I suppose.

Reply:
done with this commit
https://github.com/skuep/kicad-plugins/pull/8/commits/0ec66aee348139336e70d31d271eb7cc644a8695